contentful/apps
# feat: add error handling for selected projects that do not have protection enabled

## Purpose
When a user selects a project, that project may not be setup correctly to be displayed in our Preview. We require the project to have protection bypass enabled.

## Approach

- Add logic around checking if the protection bypass is enabled
- Add case to the error reducer

## Testing steps

1. Go to Vercel and disable protection bypass for your project
1. Go to your app's config page
1. Select the project with the disabled protection bypass
1. Verify the error message appears below the project selector
